catalog/USGS/USGS_NLCD_RELEASES_2021_REL_TCC_v2021-4.jsonnet

```diff
@@ -0,0 +1,317 @@
+    and NLCD TCC are to develop a consistent approach using the latest technology and advancements in TCC mapping to produce a 
+    "best available" map of TCC across the Conterminous United States (CONUS) and southeast Alaska, Hawaii and 
+    Puerto Rico-US Virgin Islands (OCONUS).
+
+    Model outputs include Science TCC, Science SE and NLCD TCC. Science TCC include data 
+    from 2008 through 2021. Science TCC is the raw direct model outputs. Science SE is the model standard deviation 
+    of the predicted values from all regression trees. The NLCD TCC have they have a high level of cartographic 
+    post-processing rather than simply representing spatial model predictions. 
+
+    Due to CONUS size and wide variety of ecotones, CONUS modeling was broken up into 54 480x480 km tiles. For each tile, 
+    a unique random forest model was built using 2011 fitted LandTrendr, 2011 CDL, and terrain data. All reference data 
+    that were part of the 70% available for model calibration that intersected tiles within a 5x5 window around the center 
+    tile were used to train the random forest model. That model was then applied to the center tile. 
+
+    Predictor layers for the TCC model include outputs
+    from the LandTrendr and terrain information. These
+    components are all accessed and processed using Google Earth Engine (Gorelick et al., 2017).
+
+    To produce annual composites for LandTrendr, USGS Collection 2 Landsat Tier 1 and Sentinel 2A, 
+    2B Level-1C top of atmosphere reflectance data were used. The cFmask cloud masking algorithm 
+    (Foga et al., 2017), which is an implementation of Fmask 2.0 (Zhu and Woodcock, 2012) 
+    (Landsat-only), cloudScore (Chastain et al., 2019) (Landsat-only), and s2cloudless
+    (Sentinel-Hub, 2021) (Sentinel 2-only) are used to mask clouds, while TDOM
+    (Chastain et al., 2019) is used to mask cloud shadows (Landsat and Sentinel
+    2). For LandTrendr, the annual medoid is then computed to summarize cloud
+    and cloud shadow-free values from each year into a single composite.
+
+    The composite time series is temporally segmented using LandTrendr
+    (Kennedy et al., 2010; Kennedy et al., 2018; Cohen et al., 2018).
+
+    The raw composite values, LandTrendr fitted values, pair-wise differences,
+    segment duration, change magnitude, and slope, along with elevation, slope, sine of 
+    aspect, and cosine of aspect from the 10 m USGS 3D. Elevation Program (3DEP) data 
+    (U.S. Geological Survey, 2019), are used as independent predictor variables in a Random 
+    Forest (Breiman, 2001) model.
+
+    Reference data are collected from USFS Forest Inventory and Analysis (FIA) photo-interpreted TCC data, and
+    used to make wall-to wall TCC predictions on a pixel-wise basis. 
+
+    **Additional Resources**
+
+    * Please see the [LCMS Methods Brief](https://data.fs.usda.gov/geodata/rastergateway/LCMS/LCMS_v2022-8_Methods.pdf)
```

Comment:
does this need to be a single-item list?

Reply:
Should I tab the url to the next line? Not sure what to do here

Reply:
I mean, is it ok to remove '* ' before 'Please see' ?

Reply:
I removed the * so it is not a single-item list